tests/acm_observability/test_acm_observability.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+
+pytestmark = [pytest.mark.acm_observability, pytest.mark.usefixtures("kubeadmin_token")]
```

Comment:
call the fixture where needed

Reply:
discussed in [this](https://github.com/RedHatQE/managed-services-integration-tests/pull/459#discussion_r1378506847) comment.
the testsuite is not relevant to start without the kubeadmin token

Reply:
ack

Reply:
None of the tests use it, can you try move this line to the conftest.py file and see if it work as expected?